include/dg/analysis/ReachingDefinitions/ReachingDefinitions.h

```diff
@@ -53,8 +53,12 @@ enum class RDNodeType {
         RETURN,
         // call node
         CALL,
+        // function pointer call
+        CALL_FUNCPTR,
```

Comment:
Why is this type of node needed in RDA? Pointers are resolved at this point (so we should be able to use simple CALL).